src/main/java/gov/usgs/earthquake/eids/QuakemlProductCreator.java

```diff
@@ -89,6 +89,16 @@
 	private Converter formatConverter = new Converter();
 	private boolean validate = false;
 
+	private boolean padForBase64Bug = false;
+
+	public QuakemlProductCreator() {
+		super();
+	}
+
+	public QuakemlProductCreator(boolean padForBase64Bug) {
+		this.padForBase64Bug = padForBase64Bug;
```

Comment:
Another option is to  add getter/setter methods to change the default behavior.

Reply:
Left the constructor, but added getters and setters so users can implement however they want